dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/api/RequestApi.kt

```diff
@@ -187,8 +187,19 @@ interface RequestApi {
     ): ResponseEntity<StoredDataRequest>
 
     /** A method for searching data requests based on filters.
+     * @param dataType If set, only the requests with are data type in dataType are returned
+     * @param userId If set, only the requests from this user are returned
+     * @param emailAddress If set, only the requests from users which email address partially matches emailAddress are
+     *  returned
+     * @param requestStatus If set, only the requests with a request status in requestStatus are returned
```

Comment:
Why is it necessary to have request and access status as a Set?

Reply:
In the frontend it will be possible to filter for multiple request statuses. So this set will be inserted into a SQL IN query. For consistency, we have used the same logic for the access status.

Reply:
Ok.